Cargo.lock

```diff
@@ -7332,6 +7332,7 @@ dependencies = [
  "solana-program-runtime",
  "solana-sdk",
  "solana-system-program",
+ "solana-vote",
```

Comment:
This pulls `crossbeam-channel` into `solana-svm`, which seems not ideal.
I wonder if we could hoist `ReplayVoteSender` and `ReplayVoteReceiver` somewhere higher. Or at least feature-wrap them so that `crossbeam-channel` can be optional.

Reply:
Rebased on top of #1623.